### PR TITLE
Ported some UI docker tests

### PR DESCRIPTION
**Pull request by abalakh**
Depends on SatelliteQE/airgun#273

Test results:
```python
pytest -v tests/foreman/ui_airgun/test_activationkey.py -k 'test_positive_add_docker_repo_cv or test_positive_add_docker_repo_ccv'
================================ test session starts =================================
platform darwin -- Python 3.6.4, pytest-3.6.1, py-1.6.0, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: xdist-1.22.0, services-1.2.1, mock-1.10.0, forked-0.2, env-0.6.2
collecting 28 items                                                                  2019-01-15 14:00:49 - conftest - DEBUG - BZ deselect is disabled in settings

collected 28 items / 26 deselected

tests/foreman/ui_airgun/test_activationkey.py::test_positive_add_docker_repo_cv PASSED [ 50%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_add_docker_repo_ccv PASSED [100%]

===================== 2 passed, 26 deselected in 328.45 seconds ======================

pytest -v tests/foreman/ui_airgun/test_computeresource.py -k test_positive_list_containers
================================ test session starts =================================
platform darwin -- Python 3.6.4, pytest-3.6.1, py-1.6.0, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: xdist-1.22.0, services-1.2.1, mock-1.10.0, forked-0.2, env-0.6.2
collecting 19 items                                                                  2019-01-15 14:08:11 - conftest - DEBUG - BZ deselect is disabled in settings

collected 19 items / 18 deselected

tests/foreman/ui_airgun/test_computeresource.py::test_positive_list_containers PASSED [100%]

===================== 1 passed, 18 deselected in 133.80 seconds ======================

pytest -v tests/foreman/ui_airgun/test_contentview.py -k 'test_positive_add_docker_repo or test_positive_add_docker_repos or test_positive_add_synced_docker_repo or test_positive_add_docker_repo_to_ccv or test_positive_add_docker_repos_to_ccv or test_positive_publish_with_docker_repo or test_positive_publish_with_docker_repo_composite or test_positive_publish_multiple_with_docker_repo or test_positive_publish_multiple_with_docker_repo_composite or test_positive_promote_with_docker_repo or test_positive_promote_multiple_with_docker_repo or test_positive_promote_with_docker_repo_composite or test_positive_promote_multiple_with_docker_repo_composite'
================================ test session starts =================================
platform darwin -- Python 3.6.4, pytest-3.6.1, py-1.6.0, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: xdist-1.22.0, services-1.2.1, mock-1.10.0, forked-0.2, env-0.6.2
collecting 67 items                                                                  2019-01-15 14:29:57 - conftest - DEBUG - BZ deselect is disabled in settings

collected 67 items / 54 deselected

tests/foreman/ui_airgun/test_contentview.py::test_positive_add_docker_repo PASSED [  7%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_add_docker_repos PASSED [ 15%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_add_synced_docker_repo PASSED [ 23%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_add_docker_repo_to_ccv PASSED [ 30%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_add_docker_repos_to_ccv PASSED [ 38%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_publish_with_docker_repo PASSED [ 46%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_publish_with_docker_repo_composite PASSED [ 53%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_publish_multiple_with_docker_repo PASSED [ 61%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_publish_multiple_with_docker_repo_composite PASSED [ 69%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_promote_with_docker_repo PASSED [ 76%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_promote_multiple_with_docker_repo PASSED [ 84%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_promote_with_docker_repo_composite PASSED [ 92%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_promote_multiple_with_docker_repo_composite PASSED [100%]

==================== 13 passed, 54 deselected in 2023.66 seconds =====================

pytest -v tests/foreman/ui_airgun/test_repository.py -k 'test_positive_sync_docker_via_sync_status or test_positive_delete_random_docker_repo'
================================ test session starts =================================
platform darwin -- Python 3.6.4, pytest-3.6.1, py-1.6.0, pluggy-0.6.0 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: xdist-1.22.0, services-1.2.1, mock-1.10.0, forked-0.2, env-0.6.2
collecting 19 items                                                                  2019-01-15 14:22:56 - conftest - DEBUG - BZ deselect is disabled in settings

collected 19 items / 17 deselected

tests/foreman/ui_airgun/test_repository.py::test_positive_sync_docker_via_sync_status PASSED [ 50%]
tests/foreman/ui_airgun/test_repository.py::test_positive_delete_random_docker_repo PASSED [100%]

===================== 2 passed, 17 deselected in 357.51 seconds ======================
```

### CODE REVIEW

File: tests/foreman/ui_airgun/test_activationkey.py [hunk 1]
```diff
@@ -877,6 +879,93 @@ def test_positive_remove_user(session, module_org, test_name):
...
+    cvv = content_view.read().version[0].read()
+    promote(cvv, lce.id)
+    ak_name = gen_string('alphanumeric')
+    with session:
+        session.activationkey.create({
+            'name': ak_name,
+            'lce': {lce.name: True},
+            'content_view': content_view.name,
+        })
+        ak = session.activationkey.read(ak_name)
```
Comment (ldjebran) [inline tests/foreman/ui_airgun/test_activationkey.py]: as we need only details tab, can we read only the Details tab.

File: tests/foreman/ui_airgun/test_activationkey.py [hunk 2]
```diff
@@ -877,6 +879,93 @@ def test_positive_remove_user(session, module_org, test_name):
...
+    ccvv = composite_cv.read().version[0].read()
+    promote(ccvv, lce.id)
+    ak_name = gen_string('alphanumeric')
+    with session:
+        session.activationkey.create({
+            'name': ak_name,
+            'lce': {lce.name: True},
+            'content_view': composite_cv.name,
+        })
+        ak = session.activationkey.read(ak_name)
```
Comment (ldjebran) [inline tests/foreman/ui_airgun/test_activationkey.py]: same question about reading only details tab.

File: tests/foreman/ui_airgun/test_computeresource.py [hunk 3]
```diff
@@ -205,6 +205,36 @@ def test_positive_list_resource_vms(
...
 
+@tier2
+def test_positive_list_containers(session, module_org, module_loc):
+    """Create a Docker-based Compute Resource then list its running containers.
+
+    :id: ebac82ed-5a29-4a06-8aae-bd5b08f60fba
+
+    :expectedresults: Compute Resource can be created, listed and existing
+        running instances can be listed.
+
```
Comment (ldjebran) [inline tests/foreman/ui_airgun/test_computeresource.py]: :BZ: 1466240, 1478966

File: tests/foreman/ui_airgun/test_contentview.py [hunk 4]
```diff
@@ -398,6 +407,433 @@ def test_positive_add_rh_content(session):
...
+        organization=module_org,
+    ).create()
+    repo = entities.Repository(
+        url=DOCKER_REGISTRY_HUB,
+        product=module_prod,
+        content_type=REPO_TYPE['docker'],
+    ).create()
+    with session:
+        session.contentview.add_docker_repo(content_view.name, repo.name)
+        cv = session.contentview.read(content_view.name)
```
Comment (ldjebran) [inline tests/foreman/ui_airgun/test_contentview.py]: same question can we read only "docker_repositories" from cv?

File: tests/foreman/ui_airgun/test_repository.py [hunk 5]
```diff
@@ -382,6 +385,33 @@ def test_positive_sync_custom_repo_docker(session, module_org):
         assert result['result'] == 'success'
 
 
+@tier2
+@upgrade
+def test_positive_sync_docker_via_sync_status(session, module_org):
```
Comment (ldjebran) [inline tests/foreman/ui_airgun/test_repository.py]: perhaps this test should go to test_sync.py module? what do you think?

**Comment by abalakh** [comment]
@ldjebran addressed, thank you. Now PR depends on SatelliteQE/airgun#279.
Test results:
```python
pytest -v tests/foreman/ui_airgun/test_activationkey.py -k 'test_positive_add_docker_repo_cv or test_positive_add_docker_repo_ccv'
================================ test session starts =================================
platform darwin -- Python 3.6.4, pytest-4.0.2, py-1.6.0, pluggy-0.8.1 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: xdist-1.26.1, services-1.3.1, mock-1.10.0, forked-0.2, env-0.6.2, cov-2.6.1
collecting 28 items                                                                  2019-01-29 18:09:40 - conftest - DEBUG - BZ deselect is disabled in settings

collected 28 items / 26 deselected

tests/foreman/ui_airgun/test_activationkey.py::test_positive_add_docker_repo_cv PASSED [ 50%]
tests/foreman/ui_airgun/test_activationkey.py::test_positive_add_docker_repo_ccv PASSED [100%]

=============== 2 passed, 26 deselected, 2 warnings in 177.22 seconds ================

pytest -v tests/foreman/ui_airgun/test_contentview.py -k 'test_positive_add_docker_repo or test_positive_add_docker_repos or test_positive_add_synced_docker_repo or test_positive_add_docker_repo_to_ccv or test_positive_add_docker_repos_to_ccv or test_positive_publish_with_docker_repo or test_positive_publish_with_docker_repo_composite'
================================ test session starts =================================
platform darwin -- Python 3.6.4, pytest-4.0.2, py-1.6.0, pluggy-0.8.1 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: xdist-1.26.1, services-1.3.1, mock-1.10.0, forked-0.2, env-0.6.2, cov-2.6.1
collecting ... 2019-01-29 18:29:22 - conftest - DEBUG - BZ deselect is disabled in settings

collected 67 items / 60 deselected

tests/foreman/ui_airgun/test_contentview.py::test_positive_add_docker_repo PASSED [ 14%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_add_docker_repos PASSED [ 28%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_add_synced_docker_repo PASSED [ 42%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_add_docker_repo_to_ccv PASSED [ 57%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_add_docker_repos_to_ccv PASSED [ 71%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_publish_with_docker_repo PASSED [ 85%]
tests/foreman/ui_airgun/test_contentview.py::test_positive_publish_with_docker_repo_composite PASSED [100%]

=============== 7 passed, 60 deselected, 2 warnings in 765.63 seconds ================

pytest -v tests/foreman/ui_airgun/test_sync.py -k test_positive_sync_docker_via_sync_status
================================ test session starts =================================
platform darwin -- Python 3.6.4, pytest-4.0.2, py-1.6.0, pluggy-0.8.1 -- /Users/andrii/workspace/py3a/bin/python3.6
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /Users/andrii/workspace/robottelo, inifile:
plugins: xdist-1.26.1, services-1.3.1, mock-1.10.0, forked-0.2, env-0.6.2, cov-2.6.1
collecting ... 2019-01-29 18:45:48 - conftest - DEBUG - BZ deselect is disabled in settings

collected 5 items / 4 deselected

tests/foreman/ui_airgun/test_sync.py::test_positive_sync_docker_via_sync_status PASSED [100%]

================ 1 passed, 4 deselected, 2 warnings in 219.99 seconds ================
```

**Pull request merged**